init.go

```diff
@@ -119,6 +120,7 @@ func onReady() {
 		return
 	}
 	notificationsChan <- data.NotificationEvent{Type: data.NotificationEvent_READY}
+	isReady = true
```

Comment:
Since the "ready" variable  is accessed from multi go routine the access to it should be synchronized or set/get via atomic package.